### DISCUSSION TITLE
APS Sceduler with Gunicorn and uvicorn workers

### DISCUSSION THREAD

**Discussion by airen1986** [agronholm/apscheduler#578] [Q&A]
Hi,

We have had uvicorn==0.15.0 with gunicorn==20.1.0 (uvicorn workers) where scheduled jobs were not executing, then we changed uvicorn to uvicorn==0.13.4 and installed its dependencies of uvloop and httptools, then the jobs are getting executed. Is there any compatibility matrix or are there any additional settings required for uvicorn==0.15.0 ?

Thanks

**Comment by agronholm** [comment]
I have personally never used either in production, and never combined together even at home. I don't generally endorse gevent due to its inherent problems. Perhaps someone else can answer this question?
